resources/js/Pages/Index.jsx

```diff
@@ -1,11 +1,14 @@
-import React from "react";
+import React from 'react';
+import Layout from '../Shared/Layout';
 
 const Index = () => {
   return (
     <div>
-      <h1>Index Page</h1>
+      <h3>Index Page</h3>
     </div>
   )
 };
 
+Index.layout = page => <Layout children={page} />;
```

Comment:
```suggestion
Index.layout = page => <Layout>{page}</Layout>;
```

The same `children={` is used 4 times in this PR

Passing children is almost always done by passing it into the jsx brackets. It's just a convention